app/src/main/java/com/example/calculator/MainActivity.kt

```diff
@@ -6,6 +6,7 @@ import android.os.Bundle
 import android.widget.Button
 import com.example.calculator.CalculatorOperations.*
 
+// Activity consists of 4 buttons to perform 4 different types of arithmetic operations (Addition, Subtraction, Multiplication, and Division)
```

Comment:
Try avoiding numerics ('4') as much as possible in the comments..You can replace it with 'four'